regression/tests/helpers.py

```diff
@@ -4,6 +4,19 @@
 import os
 
 
+def get_course_info():
+    """
+    Returns the course info of the course that we use for
+    the regression tests.
+    """
+    return {
```

Comment:
I think this should be an environment var or something like that. We may decide to change which course we're using, and we should be able to do that without modifying code...